control_panel_api/views.py

```diff
@@ -141,14 +140,13 @@ class AppCustomersAPIView(GenericAPIView):
     permission_classes = (IsSuperuser,)
 
     def get(self, request, *args, **kwargs):
-        instance = self.get_object()
+        app = self.get_object()
+        customers = app.get_customers()
 
-        members = Auth0().get_group_members(instance.slug)
-
-        if members is None:
+        if not customers:
```

Comment:
Isn't this true when there are no customers for an app?
Shouldn't this respond with a `200 OK` and empty list of customers when there are no customers instead?